.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,103 @@
+# Original source: https://github.com/AcalaNetwork/Acala/blob/master/.github/workflows/benchmark.yml
+
+name: Benchmark
+on:
+  issue_comment:
+    types: [created]
+permissions:
+  pull-requests: write
+  contents: write
+jobs:
+  benchmark:
+    name: Benchmark
+    if: ${{ github.event.issue.pull_request && contains(fromJson('["MEMBER", "OWNER"]'), github.event.comment.author_association) && startsWith(github.event.comment.body, '/bench ') }}
+    runs-on: [self-hosted, linux]
+    steps:
+      - uses: actions/github-script@v6
+        name: Get PR branch
+        id: issue
+        with:
+          script: |
+            const pr = context.payload.issue.number
+            const data = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: pr
+            })
+            return {
+              ref: data.data.head.ref,
+              sha: data.data.head.sha,
+            }
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          ref: ${{ fromJson(steps.issue.outputs.result).sha }}
+      - uses: actions/github-script@v6
+        name: Prepare command
+        id: command
+        with:
+          result-encoding: string
+          script: |
+            const [, , cmd, ...args] = context.payload.comment.body.split(/\W+/)
+            switch (cmd) {
+              case 'runtime': {
+                const [runtime, pallet] = args
+                if (!['kintsugi', 'kintsugi-dev'].includes(runtime)) throw new Error('Invalid runtime')
```

Comment:
We will also need to run this for `interlay` and (for now) the testnet runtimes.